content/source/docs/enterprise/migrate/index.html.md

```diff
@@ -27,74 +28,77 @@ Make sure you have all of the following:
 - A TFE user account which is a member of your organization's owners team, so you can create workspaces.
 - A [user API token][user-token] for your TFE user account. (Organization and team tokens will not work; the token must be associated with an individual user.)
 
-    In your shell, set an `ATLAS_TOKEN` environment variable with your API token as the value.
+    In your shell, set a `TFE_TOKEN` environment variable with your API token as the value.
 
     ``` bash
-    export ATLAS_TOKEN=<USER TOKEN>
+    export TFE_TOKEN=<USER TOKEN>
```

Comment:
Going forward we're consolidating credential storage here https://www.terraform.io/docs/commands/cli-config.html#credentials - the ENV variable actually won't work in this case, so we should point users to the config file to set their API token.